package/packer/proxmox/cloud/README.md

```diff
@@ -0,0 +1 @@
+# Template k3os to Proxmox VE
```

Comment:
Please provide a sample build command in the readme, e.g.
```bash
packer build -var-file=variables.json k3os-cloud.json
```

See https://github.com/rancher/k3os/blob/master/package/packer/vagrant/README.md as an example.

Reply:
Done.